### DISCUSSION TITLE
Contrast Effect by kvantum does not affect window decorations.

### DISCUSSION THREAD

**Discussion by bluee111** [tsujan/Kvantum#802] [General]
Hello friends, does anyone know how to make the Contrast Effect of Kvantum apply to window decorations ?
The Contrast Effect is a good solution for those very bright white backgrounds, but if I put Contrast Effect the color of the window decorations does not match in any way. any idea?
https://i.ibb.co/y6Sm35y/2022-06-17-19-20.png

**Comment by tsujan** [comment]
Window decoration is controlled by the window manager (KWin in KDE) and its themes, not by the Qt widget style (Kvantum).

You need a KWin window decoration that supports contrast setting. I don't know of any, but there may be some.

You might want to ask about these things in KDE forums. Kvantum doesn't depend on KDE.
